CHANGELOG.md

```diff
@@ -35,6 +35,9 @@ Here is a template for new release sections
 - Add check for the capability of asset capacities in an energy system to fulfill the maximum demand in `C1.check_energy_system_can_fulfill_max_demand()` (#824)
 - Add tests for `C1.check_energy_system_can_fulfill_max_demand()`, including pseudo-benchmark test in `test_C1` (#824)
 - Zenodo badge on README (#850)
+- Introduce new parameter `maximumAddCap` where `maximumAddCap=maximumCap-installedCap` in `C0.process_maximum_cap_constraint()` (#848)
+- Implement definition for `maximumAddCap` in `docs/MVS_parameters_list.csv` (#848)
```

Comment:
`MAXIMUM_ADD_CAP` is something that is added in the pre-processing in `C0`, right? As such, I am not sure that it has to be in the `MVS_parameters_list.csv`, as it is not an input parameter that the user can set. @Bachibouzouk did I understand the use of `MVS_parameters_list.csv` correctly?